.gitignore

```diff
@@ -6,5 +6,6 @@ build/
 node_modules/
 npm-debug.log
 rethinkdb_data/
+rethinkdb_dump_*
```

Comment:
This is the prefix of database backups created with `rethinkdb dump`. Just found that out.